route-parser/index.js

```diff
@@ -1,2 +1,2 @@
-export { default } from 'route-parser';
-export * from 'route-parser';
+//export { default } from 'route-parser';
+//export * from 'route-parser';
```

Comment:
Ahh, this file got in by mistake. I commented this out locally because it was giving me exception. Not sure who has added this library. Removing this file from this PR.